src/Offer/Security/SearchQueryFactory.php

```diff
@@ -1,26 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace CultuurNet\UDB3\Offer\Security;
-
-use ValueObjects\StringLiteral\StringLiteral;
-
-/**
- * Implementation of the search query factory for SAPI3.
- */
-class SearchQueryFactory extends SearchQueryFactoryBase
```

Comment:
To be sure I understand correctly, this was unrelated but you spotted this by coincidence? Nice find! 🕵️

Reply:
Yeah, it had dependencies on the Query VO but it wasn't used at all